templates/media/media--image.html.twig

```diff
@@ -0,0 +1,49 @@
+{#
+/**
+ * @file
+ * Theme override to display an image media item with WA-compliant description.
+ * ~ When field_aria_description has value, this template:
+ * ~~ Adds aria-describedby="description-[MID]" to field_media_image <img> tag.
+ * ~~ Renders field_aria_description inside div#description-[MID].sr-only.
+ * See cwd_base/media/media.html.twig for available variables and other info.
+ *
+ * @see template_preprocess_media()
+ *
+ * @ingroup themeable
+ */
+#}
+{{ title_suffix.contextual_links }}
+{% if content %}
+
+  {% if media.field_aria_description.0.value -%}
+    {% if media.field_media_image.0.value -%}
+
+      {####### variable setup #######}
+      {% set media_id = media.mid.0.value %}
+        {% set describedby = 'description-' ~ media_id %}
+
+        {% set compleximg_alt = media.field_media_image[0].alt ?? '' %}
+        {% set compleximg_url = media.field_media_image.entity.uri.value %}
+        {% if content.field_media_image[0]['#image_style'] %}
+          {# If the current view mode uses an image style, use that image style. #}
+          {% set style_name = content.field_media_image[0]['#image_style'] %}
+          {% set compleximg_url = file_url(compleximg_url|image_style(style_name)) %}
+        {% else -%}
+          {% set compleximg_url = file_url(compleximg_url) %}
+        {%- endif %}
+
+      {####### output #######}
+      <img src="{{ compleximg_url }}" alt="{{ compleximg_alt }}" aria-describedby="{{ describedby }}" />
+      <div id="description-{{ media_id }}" class="sr-only">
+        {{ media.field_aria_description|view }}
+      </div>
```

Comment:
FULL cough-cough summary of changes in this suggestion, pls lmk if I missed anything:
* create `describedby` in one step
* use file_uri filter to get URI right from "media" (so to speak), instead of getting it from `media.field_media_image.entity.uri.value`
* set fallback style_name value of '' (for use with [drupal_image twig_tweak function](https://www.drupal.org/docs/contributed-modules/twig-tweak/cheat-sheet#s-drupal-image), which Eric is okay with in place of an actual `<img>` :stuck_out_tongue:)

Looks tidy to me, I'll give it a try on the multidev.  Thanks!